ouroboros-consensus-byron/test/Test/Consensus/Byron/Ledger/Golden.hs

```diff
@@ -4,21 +4,36 @@
 module Test.Consensus.Byron.Ledger.Golden (tests) where
 
 import           Codec.CBOR.Decoding (Decoder)
-import           Codec.CBOR.FlatTerm (TermToken (..))
+import           Codec.CBOR.FlatTerm (FlatTerm, TermToken (..))
 import           Codec.CBOR.Read (deserialiseFromBytes)
 import           Control.Monad.Except (runExcept)
 import qualified Data.ByteString.Lazy as Lazy
 import qualified Data.ByteString.Lazy.Char8 as Lazy8
+import qualified Data.Map as M
 import qualified Data.Sequence.Strict as Seq
+import qualified Data.Set as Set
 
 import           Cardano.Binary (toCBOR)
 import qualified Cardano.Chain.Block as CC.Block
+import           Cardano.Chain.Byron.API
+import           Cardano.Chain.Common (LovelaceError (..))
+import           Cardano.Chain.Genesis (configGenesisHeaderHash)
+import           Cardano.Chain.Update (ApplicationName (..))
 import qualified Cardano.Chain.Update.Validation.Interface as CC.UPI
+import           Cardano.Chain.Update.Validation.Endorsement (
+                     CandidateProtocolUpdate (..), Endorsement (..))
+import           Cardano.Chain.Update.Validation.Registration
+                     (ApplicationVersions, ProtocolUpdateProposals,
+                     SoftwareUpdateProposals)
+import           Cardano.Chain.UTxO (annotateTxAux, TxValidationError (..),
+                     UTxOValidationError (..))
```

Comment:
I prefer using `CC` (or `CC.<something more specific>`) to make it clear that these types come from `cardano-ledger`.